PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -256,7 +258,10 @@
     )
 for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016:
   modifier.toModify( jetTable.variables, jetId = Var("userInt('tightIdLepVeto')*4+userInt('tightId')*2+userInt('looseId')",int,doc="Jet ID flags bit1 is loose, bit2 is tight, bit3 is tightLepVeto"))
+run2_nanoAOD_102Xv1.toModify( jetTable.variables, puIdDisc = Var("userFloat('puId102XDisc')",float,doc="Pileup ID discriminant with 102X (2018) training",precision=10) )
+run2_nanoAOD_102Xv1.toModify( jetTable.variables, puId = Var("userInt('pileupJetId:fullId')",int,doc="Pileup ID flags for pre-UL trainings") )
```

Comment:
after #32960 it means that I should leave `pu102XDisc` and `pileupJetId:fullId` as it wasm and include:
```
run2_nanoAOD_devel.toModify( puIdDisc = Var("userFloat('puId106XUL18Disc')", float,doc="Pileup ID discriminant with 106X (2018) training",precision=10) )
run2_nanoAOD_devel.toModify( puId = Var("userInt('puId106XUL18Id')", int,doc="Pileup ID flags with 106X (2018) training") )
```
Is this correct @mariadalfonso ?